src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -16,8 +16,9 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Separate names by commas (,)"
+            + "Parameters: KEYWORD, [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " alice, bob, charlie";
```

Comment:
May I check why there is a need to change this to using commas after each word?

Reply:
Okay I understood by your example 
"If Hustlebook contains "David Li" and "David Tan", the command find david tan should only show "David Tan" but not both of them". 

Then can I check is it possible to execute `find David Li, David Tan, Bob, alice` and expect both David Tan and David Li to appear in the results along with any names containing bob or alice? In other words, is it possible that`KEYWORD` is not just one word.

Reply:
yes that's how it works

Reply:
Nice!